ocaml/xapi/xapi_globs.ml

```diff
@@ -715,6 +715,7 @@ let sm_dir = ref "/opt/xensource/sm"
 let web_dir = ref "/opt/xensource/www"
 
 let cluster_stack_root = ref "/usr/libexec/xapi/cluster-stack"
+let cluster_stack_compatible_srs = ref "ha_statefile_srs"
```

Comment:
Unless we intend to make this configurable you can just put this inside `xha_scripts.ml` like the other script names are as a regular string, not a ref.

Reply:
Sure, will update now (will also need to update the other commit)